### PR TITLE
update coredns and config for 1.6.2

### PR DESCRIPTION
**Pull request by rmenn**
Fixes : #1059

### CODE REVIEW

File: pkg/asset/internal/templates.go [hunk 1]
```diff
@@ -806,13 +806,16 @@ data:
     .:53 {
         errors
         health
-        kubernetes cluster.local {{ .ServiceCIDR }} {
+        ready
```
Comment (andrewrynhard) [inline pkg/asset/internal/templates.go]: Do we need to update the liveness and readiness probes to make use of this?

File: pkg/asset/internal/templates.go [hunk 2]
```diff
@@ -806,13 +806,16 @@ data:
...
+        log . {
+            class error
```
Comment (andrewrynhard) [inline pkg/asset/internal/templates.go]: Log errors only?

Comment (rmenn) [reply pkg/asset/internal/templates.go]: yes

File: pkg/asset/internal/templates.go (hunk 1 above)
Comment (rmenn) [reply pkg/asset/internal/templates.go]: isnt it better to have that in a separate PR ?

**Pull request merged**